packages/components/src/TokenTutorial/TokenTutorialLanding.tsx

```diff
@@ -93,7 +90,7 @@ export class TokenTutorialLanding extends React.Component<TokenTutorialLandingPr
                 <TutorialProgressText questions={topic.questions.length - lastSlideIdx} />
                 <TutorialLandingProgressBars>
                   {topic.questions.map((question, questionIdx) => {
-                    if (lastSlideIdx > 0 && questionIdx <= lastSlideIdx) {
+                    if (lastSlideIdx > 0 && questionIdx <= lastSlideIdx - 1) {
```

Comment:
"There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors."